test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/initialgainorloss/InitialGainOrLossControllerSpec.scala

```diff
@@ -133,8 +133,11 @@ class InitialGainOrLossControllerSpec
           val draftReturn = sample[DraftSingleDisposalReturn]
             .copy(initialGainOrLoss = Some(AmountInPence(300L)))
 
-          val fillingOutReturn      = sample[FillingOutReturn].copy(draftReturn = draftReturn)
-          val trustFillingOutReturn = fillingOutReturn.copy(subscribedDetails   = generateTrustSubscribedDetails())
+          val fillingOutReturn = sample[FillingOutReturn].copy(draftReturn = draftReturn)
+          val trustFillingOutReturn = fillingOutReturn.copy(
+            subscribedDetails = generateTrustSubscribedDetails(),
+            draftReturn       = draftReturn.copy(triageAnswers = generateTriageAnswersWithSelf())
```

Comment:
for a trust, the individualUserType should be `None` for the most realistic setup